docs/contributing.rst

```diff
@@ -86,12 +86,42 @@ modules only, rather than individual classes or functions.
 Testing
 ^^^^^^^
 
-Twine is tested against Python versions 3.6, 3.7, and 3.8. To run these tests
-locally, you will need these versions of Python installed on your machine.
+We use `pytest`_ for writing and running tests.
```

Comment:
I try to use imperative voice instead of first-person plural.

```suggestion
Twine uses `pytest`_ for writing and running tests.
```

Reply:
I was wondering about that; I saw `we` used in a few other spots, so I normalized on that, but I'm happy to switch them all.

Reply:
I see that voice was previously used and probably could be retained for the purposes of this change.

Reply:
Leaving as-is for now.